src/grpcbox_stream.erl

```diff
@@ -61,15 +62,19 @@
 
 -type t() :: #state{}.
 
--type grpc_status_message() :: unicode:chardata().
+-type grpc_status_message() :: unicode:unicode_binary().
 -type grpc_status() :: 0..16.
 -type http_status() :: integer().
--type grpc_error() :: {grpc_status(), grpc_status_message()}.
+-type grpc_error() :: {binary(), % containing a grpc_status() value as text
```

Comment:
```suggestion
-type grpc_error() :: {unicode:unicode_binary(), % containing a grpc_status() value as text
```

Assuming this will always be a utf8 binary string?

Reply:
Done (yes, I suppose it will, although I admit I haven't read enough http2 spec to know)